spec/integration/manifest.spec.yml

```diff
@@ -90,6 +90,7 @@ spec:
           - web_application_firewall
           - websphere
           - workplace_search
+          - cloudsecurity_cdr
```

Comment:
Nit. Please add it in alphabetical order.

Reply:
done.